code/core/src/types/modules/core-common.ts

```diff
@@ -336,6 +342,30 @@ export interface TagOptions {
 
 export type TagsOptions = Record<Tag, Partial<TagOptions>>;
 
+interface Features extends LegacyFeatureFlags {
```

Comment:
Currently, we are not doing anything with the new `features` config. We want to disable our internal addons if these flags are set to `false`. Do you want to do this in a separate PR?